frontend/express/public/core/events/javascripts/countly.details.views.js

```diff
@@ -333,16 +376,16 @@
             eventKey: eventKey && eventKey !== "undefined" ? eventKey : undefined
         };
         var EventAllView = getAllEventsView();
-        AllEventsView.params = params;
+        EventAllView.params = params;
         this.renderWhenReady(EventAllView);
     });
 
     app.route("/analytics/events", "all-events", function() {
         var params = {
-            eventKey: undefined
+            eventKey: localStorage.getItem("eventKey") || undefined
```

Comment:
what is the purpose of setting eventKey in params here?

Reply:
you should remove this from here and set it here perhaps - https://github.com/Countly/countly-server/pull/1972#discussion_r699051064

Reply:
as per our discussion this is expected in the upcoming pr.
keeping a note.